x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -156,6 +160,7 @@ public static NamedClusterPrivilege resolve(String name) {
         if (fixedPrivilege != null) {
             return fixedPrivilege;
         }
+        logger.error("failed to resolve cluster privilege [" + name + "].");
```

Comment:
I don't think this message is as helpful as it could be.
What do we expect someone to do if they read this? Why is it an `ERROR` ?